stock_exchange/apps/stock/models.py

```diff
@@ -1,4 +1,40 @@
+from datetime import date
+
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from stock_exchange.apps.users.models import User
+from .choices import MARKET_OPTIONS
+
+class Currency(models.Model):
+    # Storage name of currency
+    name = models.CharField(_('Currency name'), max_length = 50)
+
+    # Storage cost in $ for currency
+    current_value = models.DecimalField(_('Actual rate'), max_digits = 30,
+        decimal_places = 5)
+
+    # Storage description of currency
+    description = models.TextField(_('Description'), blank = True, null = True)
+    
+    def __str__(self):
+        return '{}'.format(self.name)
+
+
+class Transactions(models.Model):
```

Comment:
I didn't see this earlier but name of this model should be `Transaction` not `Transactions`